File: SofiForce.Web/ClientApp/src/app/core/Models/ListModels/BusinessUnitListModel.ts
export interface BusinessUnitListModel {
    businessUnitId: number | null;
    branchId: number | null;
    businessUnitCode: string;
    businessUnitName: string;
    icon: string;
    color: string;
    displayOrder: number | null;
    isActive: boolean | null;
    canEdit: boolean | null;
    canDelete: boolean | null;
    branchCode: string;
    branchName: string;
}

File: SofiForce.Models/Models/ListModels/BusinessUnitListModel.cs
using System;
using System.Collections.Generic;

namespace Models
{
    public class BusinessUnitListModel
    {


		public Int32? BusinessUnitId { get; set; }
		public Int32? BranchId { get; set; }
		public string BusinessUnitCode { get; set; }
		public string BusinessUnitName { get; set; }
		public string Icon { get; set; }
		public string Color { get; set; }
		public Int32? DisplayOrder { get; set; }
		public bool? IsActive { get; set; }
		public bool? CanEdit { get; set; }
		public bool? CanDelete { get; set; }
		public string BranchCode { get; set; }
		public string BranchName { get; set; }


	}
}
